### DISCUSSION TITLE
Edit link?

### DISCUSSION THREAD

**Discussion by davidearl** [SVG-Edit/svgedit#774] [New feature request]
How do you edit a link once you have added on to an object? In the demo the link button is only visible for new objects not for existing ones, and I can't see the existing link in the UI anywhere to edit it.

**Comment by davidearl** [comment]
OK, I see what is happening. Rather than pressing the button again, you intend there to be a box in the top row where the link is displayed and editable.

However, this is only visible if the browser window is more than about 1700px wide (for a rectangle, a bit less for ellipse). Any less (and that's most browser windows I'd venture, even on big screens I certainly don't keep it maximised), and it tucks itself invisibly under the ruler. There is a teensy-weensy scroll bar appears far top right of which only the arrows are visible, which will scroll it onto view. However, that's essentially invisible unless you know it's there, and the panel is effectively undiscoverable.

I really think the height of the panel should grow to accommodate overflowed items, but all the positioning seems to be absolute, so that's been made very difficult to manage.

This control (and any others which cause the same behaviour) is essentially impossible to find, so I really think it counts as a bug.

**Comment by brettz9** [comment]
I don't know if anything has changed since you reported this issue (I believe icons are now smaller), but I'm labeling as an enhancement for now.
